### PR TITLE
tp.begin and tp.end operators

### PR DESCRIPTION
**Pull request by achoum**
Based on https://github.com/google/temporian/pull/147

See operator design doc: https://docs.google.com/document/d/1ew_Fh-ikimrHRX75YCwjVHmL_BghIp497HOspswo-N0/edit?resourcekey=0-BgPpf9qM-8ZeOT4Bhb26cA

### CODE REVIEW

**Review by ianspektor — APPROVED** [review]
Thanks for all the changes!

One more thing: since this PR addresses most of yesterday's action items, maybe it could make sense to also replace all occurrences of `evtset` with `evset`? Up to you, we can address it in a follow up one too.

File: temporian/__init__.py [hunk 1]
```diff
@@ -23,51 +23,76 @@
...
-from temporian.core.data import dtype
-from temporian.core.data import node
-from temporian.core.data import schema
+# # Load packages privately
+# from temporian.core import serialization as _serialization
+# from temporian.core.data import dtype as _dtype
+# from temporian.core.data import node as _node
+# from temporian.core.data import schema as _schema
+# from temporian.core.data import duration as _duration
+# from temporian.core import evaluation as _evaluation
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/__init__.py]: Yes.
*Done

**Review by rstz — APPROVED** [review]
LGTM

**Review by rstz — APPROVED** [review]
LGTM

**Pull request merged**